boms/integration-tests/src/test/java/com/google/cloud/MaximumLinkageErrorsTest.java

```diff
@@ -17,19 +17,23 @@
 
 package com.google.cloud;
 
+import static com.google.cloud.tools.opensource.dependencies.RepositoryUtility.newRepositorySystem;
```

Comment:
This method probably shouldn't be a static import since it's an uncommon method and no one will know where it comes from, unlike assertTrue, for example.

Reply:
Updated.